data/api/client-server/report_content.yaml

```diff
@@ -62,7 +125,7 @@ paths:
                     and 0 is inoffensive.
                 reason:
                   type: string
-                  description: The reason the content is being reported. May be blank.
+                  description: The reason the content is being reported.
```

Comment:
I have removed this as proposed in [MSC2414](https://github.com/matrix-org/matrix-spec-proposals/pull/2414) and thought it aligns well enough with the rest of this pull request to sneak in. 😇